app/src/main/python/gogdl/cli.py

```
@@ -0,0 +1,177 @@
+            # Exchange authorization code for access token
+            logger.info("Exchanging authorization code for access token...")
+            
+            token_data = {
+                "client_id": GOG_CLIENT_ID,
+                "client_secret": GOG_CLIENT_SECRET,
+                "grant_type": "authorization_code",
+                "code": arguments.code,
+                "redirect_uri": "https://embed.gog.com/on_login_success?origin=client"
+            }
+            
+            response = requests.post(GOG_TOKEN_URL, data=token_data)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)